pom.xml

```diff
@@ -22,6 +22,7 @@
         <groupId>no.nav.eux</groupId>
         <artifactId>parent-pom</artifactId>
         <version>1.0.13</version>
+        <relativePath/>
```

Comment:
er det en grunn til å legge på denne?

Reply:
IntelliJ klager. Default verdi for relativePath er "../pom.xml" men der ligger ikke det forventede parent-prosjektet.

The relative path of the parent pom.xml file within the check out. If not specified, it defaults to ../pom.xml. Maven looks for the parent POM first in this location on the filesystem, then the local repository, and lastly in the remote repo. relativePath allows you to select a different location, for example when your structure is flat, or deeper without an intermediate parent POM. However, the group ID, artifact ID and version are still required, and must match the file in the location given or it will revert to the repository for the POM. This feature is only for enhancing the development in a local checkout of that project. Set the value to an empty string in case you want to disable the feature and always resolve the parent POM from the repositories.
Default value is: ../pom.xml